src/main/resources/sql/changelog/2017-10-12_Remove_Vinita_Terrace_Municipality.sql

```diff
@@ -0,0 +1,6 @@
+SET FOREIGN_KEY_CHECKS=0;
```

Comment:
We should never do this. Foreign keys exist for a reason.

Instead delete in the order that satisfies the relational constraints